config/collector.yaml

```diff
@@ -16,9 +16,9 @@ exporters:
   googlecloud:
     log:
       default_log_name: opentelemetry-collector
-    user_agent: Google-Cloud-OTLP manifests:0.1.0 otel/opentelemetry-collector-contrib:0.118.0
+    user_agent: Google-Cloud-OTLP manifests:0.3.0 google-built-otel/opentelemetry-collector-contrib:0.121.0
```

Comment:
Discussed offline; we are going to change this User Agent to more closely match the user agent the Google-Built OTel Collector will naturally produce.

Reply:
Any reason for the jump from 0.1.0 &rarr; 0.3.0 ?

Reply:
In the description of https://github.com/GoogleCloudPlatform/otlp-k8s-ingest/pull/43 it says the `manifest` will be changed to `0.2.0`

Reply:
No, we requested changes on that PR to move it back to v0.1.0. 

So even if it gets merged, manifest version will not be updated.

Reply:
Ah, ok will move it back down to 0.1.0

Reply:
I think for this PR, we can move it 0.2.0 since we are switching images and updating user-agent string.

So this change would be a good candidate for release.

Reply:
Changed User Agent to `Google-Cloud-OTLP manifests:0.2.0 OpenTelemetry Collector Built By Google/0.121.0 (linux/amd64)`